core/src/main/scala/org/apache/spark/deploy/ExternalShuffleService.scala

```diff
@@ -37,15 +39,17 @@ import org.apache.spark.util.Utils
  * Optionally requires SASL authentication in order to read. See [[SecurityManager]].
  */
 private[deploy]
-class ExternalShuffleService(sparkConf: SparkConf, securityManager: SecurityManager)
+class ExternalShuffleService(
+    sparkConf: SparkConf,
+    securityManager: SecurityManager,
+    transportConf: TransportConf,
```

Comment:
You don't need to pass this in here. In fact, do we even use this, since we create another one from `sparkConf` down there?